easybuild/easyblocks/i/itac.py

```diff
@@ -62,11 +63,18 @@ def install_step(self):
         """
 
         if LooseVersion(self.version) >= LooseVersion('8.1'):
-            super(EB_itac, self).install_step(silent_cfg_names_map=None)
+            silent_cfg_names_map = {}
+            if LooseVersion(self.version) >= LooseVersion('9.0'):
+                # To generate the correct itacvars.sh (called by traceanalyzer) we need to change the default install dir
+                silent_cfg_names_map = {
+                    'install_dir': install_path(),
```

Comment:
HMNS with `EASYBUILD_FIXED_INSTALLDIR_NAMING_SCHEME` would still work, but I get your point. Patching `itacvars.sh` might be safer, but I have to admit that I dislike to do something (install in `installdir/itac/version`) to undo it later (`move_after_install` and patch `itacvars.sh`). Doesn't look right.